TuringAdvisoryGroup/tryrolljs
# Add select component

## What's done

- Add Select & Popover components
- Replace TextField with Input component
- Remove react-native-paper dependency
- Add iOS & Android storybooks

## How to test

1. `yarn start`.
2. `yarn start:native` => `yarn ios`
3. `yarn start:native` => `yarn android`
4. Ensure that `Select` works for all environments.

## Tasks


## Demo (screenshots, recordings, etc.)

<img width="477" alt="Screen Shot 2022-10-24 at 13 31 46" src="https://user-images.githubusercontent.com/17337276/197516394-3d84a610-11d9-4a21-83aa-3d66ff3bc588.png">
<img width="564" alt="Screen Shot 2022-10-24 at 13 31 48" src="https://user-images.githubusercontent.com/17337276/197516397-9b6baa8a-0fd0-4638-93b9-89ce998a70c0.png">
<img width="946" alt="Screen Shot 2022-10-24 at 13 32 04" src="https://user-images.githubusercontent.com/17337276/197516400-03ff722d-6120-493c-882e-b9495310a3a2.png">
